packages/editor/src/client/panel/peerProfileEditPanel.tsx

```diff
@@ -33,7 +33,10 @@ export function PeerInfoEditPanel({onClose, onSave}: ImageEditPanelProps) {
   const [name, setName] = useState('')
   const [themeColor, setThemeColor] = useState('')
   const [callToActionText, setCallToActionText] = useState<RichTextBlockValue>(createDefaultValue())
-  const [callToActionURL, setCallToActionURL] = useState('')
+  const [callToActionTextURL, setCallToActionTextURL] = useState('')
+  const [callToActionImageID, setCallToActionImageID] = useState<PeerProfileImage>()
```

Comment:
I'm not sure if the `PeerProfileImage` is the right type. It's used for the peer profile and you have here just a normal image.